src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/Cache.java

```diff
@@ -78,7 +77,7 @@ public Cache(
         this.cachedResponseBufferPool = bufferPool.duplicate();
         this.responseBufferPool = bufferPool.duplicate();
         this.subscriberBufferPool = bufferPool.duplicate();
-        this.cachedEntries = new Int2ObjectHashMap<>();
+        this.cachedEntries = new Int2CacheHashMapWithLRUEviction();
```

Comment:
Does `org.agrona.collections.IntLruCache` work here instead?

Reply:
IntLruCache has a fixed container size.  Unfortunately, I'm not sure what fixed size to initialize it as because there is not a 1 to 1 ratio of cached responses to slots.  Which is why I created Int2CacheHashMapWithLRUEviction

Reply:
Okay.